From 2f60bbeae2484264e47cca5845e1b41e0f457ed0 Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Wed, 26 Oct 2016 20:16:59 +0000
Subject: [PATCH 15/21] Revert "scripts: Create directory and link to
 resolv.conf"

This reverts commit 598248402eb82ad870ca8ac4b1a4e5f6bec948dc.
---
 scripts/connman.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/scripts/connman.in b/scripts/connman.in
index 0b9f63c..1692b95 100644
--- a/scripts/connman.in
+++ b/scripts/connman.in
@@ -9,11 +9,6 @@ if [ -f @sysconfdir@/default/connman ] ; then
 	. @sysconfdir@/default/connman
 fi
 
-if [ "CONNMAN_RUNSTATEDIR_RESOLVCONF" != "no" ] ; then
-    mkdir -p @runstatedir@/connman
-    ln -sf @runstatedir@/connman/resolv.conf /etc/
-fi
-
 set -e
 
 do_start() {
-- 
2.1.4

